cdc/owner/changefeed.go

```diff
@@ -128,6 +132,17 @@ func (c *changefeed) Tick(ctx cdcContext.Context, state *orchestrator.Changefeed
 			Message: err.Error(),
 		})
 		c.releaseResources(ctx)
+	} else {
+		// c.initialize(ctx)
+		costTime := time.Since(startTime)
```

Comment:
May be it is better to rewrite the whole `if block` as:
```
err := c.tick()
costTime = time.Since(startTime)
// warn log here
if err != nil {
    ...
}
... 
```